eng/pipelines/dotnet-buildtools-prereqs-ubuntu19.yml

```
@@ -4,13 +4,13 @@ trigger:
     - master
   paths:
     include:
-    - src/ubuntu/19.04/*
+    - src/ubuntu/*
 pr: none
 
 variables:
 - template: variables/common.yml
 - name: imageBuilder.pathArgs
-  value: --path 'src/ubuntu/19.04*'
+  value: --path 'src/ubuntu/*'
```

There are already YAML pipelines defined for 14, 16, and 18.  This should explicitly only include 19*.

```suggestion
  value: --path 'src/ubuntu/19*'
```